config/webpack/loaders/vue.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'module' is not defined  no-undef